cmake/Macros.cmake

```diff
@@ -28,6 +28,9 @@ endfunction()
 #                      [STATIC]) # Always create a static library and ignore BUILD_SHARED_LIBS
 macro(sfml_add_library target)
 
+    # add convenience variable
+    set(TARGET_NAME "${target}")
```

Comment:
I suggest moving this just below the call to configure_file() as it's the only place where it's used

Reply:
Needs to be before configure_file, otherwise it won't propagate through the
resource.rc.in file.

Reply:
My bad, I meant "just above" :D